static/js/jira-manual-app-creation.js

```diff
@@ -1,79 +1,130 @@
+/* globals $, AP */
 const params = new URLSearchParams(window.location.search.substring(1));
 const jiraHost = params.get("xdm_e");
 
 const openChildWindow = (url) => {
-  const child = window.open(url);
-  const interval = setInterval(function () {
-    if (child.closed) {
-      clearInterval(interval);
-      AP.navigator.go(
-        'addonmodule',
-        {
-          moduleKey: "github-post-install-page"
-        }
-      );
-    }
-  }, 1000);
-
-  return child;
+	const child = window.open(url);
```

Comment:
No idea why the indentation looks like this here. Fine in my IDE 🤷

Reply:
Switch to vim!